iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/Networks.java

```diff
@@ -241,59 +236,64 @@ private static void addDisonnectedShunt(Terminal.BusBreakerView view, double q,
-        table.addCell("Generator count");
-        table.addCell(Integer.toString(balanceMainCC.connectedGenerators.size()));
-        table.addCell(Integer.toString(balanceMainCC.disconnectedGenerators.size()));
-        table.addCell(Integer.toString(balanceOtherCC.connectedGenerators.size()));
-        table.addCell(Integer.toString(balanceOtherCC.disconnectedGenerators.size()));
-        table.addCell("Max generation (MW)");
-        table.addCell(Double.toString(balanceMainCC.connectedMaxGeneration));
-        table.addCell(Double.toString(balanceMainCC.disconnectedMaxGeneration));
-        table.addCell(Double.toString(balanceOtherCC.connectedMaxGeneration));
-        table.addCell(Double.toString(balanceOtherCC.disconnectedMaxGeneration));
-        table.addCell("Generation (MW)");
-        table.addCell(Double.toString(balanceMainCC.connectedGeneration));
-        table.addCell(Double.toString(balanceMainCC.disconnectedGeneration));
-        table.addCell(Double.toString(balanceOtherCC.connectedGeneration));
-        table.addCell(Double.toString(balanceOtherCC.disconnectedGeneration));
-        table.addCell("Shunt at nom V (MVar)");
-        table.addCell(Double.toString(balanceMainCC.connectedShuntPositiveVolume) + " " +
-                Double.toString(balanceMainCC.connectedShuntNegativeVolume) +
-                " (" + Integer.toString(balanceMainCC.connectedShunts.size()) + ")");
-        table.addCell(Double.toString(balanceMainCC.disconnectedShuntPositiveVolume) + " " +
-                Double.toString(balanceMainCC.disconnectedShuntNegativeVolume) +
-                " (" + Integer.toString(balanceMainCC.disconnectedShunts.size()) + ")");
-        table.addCell(Double.toString(balanceOtherCC.connectedShuntPositiveVolume) + " " +
-                Double.toString(balanceOtherCC.connectedShuntNegativeVolume) +
-                " (" + Integer.toString(balanceOtherCC.connectedShunts.size()) + ")");
-        table.addCell(Double.toString(balanceOtherCC.disconnectedShuntPositiveVolume) + " " +
-                Double.toString(balanceOtherCC.disconnectedShuntNegativeVolume) +
-                " (" + Integer.toString(balanceOtherCC.disconnectedShunts.size()) + ")");
-        return table;
+    private static Writer writeInTable(ConnectedPower balanceMainCC, ConnectedPower balanceOtherCC) {
+        try (Writer myWriter = new StringWriter();
+             AbstractTableFormatter formatter = new AsciiTableFormatter(myWriter, "myFormatter", 5)) {
+
+            formatter.writeCell("Bus count").
+                    writeCellWithColspan("Main CC connected/disconnected", 2).
+                    writeCellWithColspan("Others CC connected/disconnected", 2);
+            formatter.writeCell("Bus count").
+                    writeCellWithColspan(Integer.toString(balanceMainCC.busCount), 2).
+                    writeCellWithColspan(Integer.toString(balanceOtherCC.busCount), 2);
```

Comment:
I think there is an issue here: you write this block 2 times

Reply:
We have 5 columns (1+2+2) :  there's no issue here.